Common Programming Concepts/Basic Data Types/Intro/task.md

```diff
@@ -8,6 +8,26 @@ Keep in mind that Rust is a _statically typed language_, which means that it mus
 let guess: u32 = "42".parse().expect("Not a number!");
 ```
 
+<div class="hint" title="Code explanation">
+
+  Although the example is focused on type annotation, it uses the `parse` and `expect` methods. 
+  We won't pay much attention to them now, but for a general understanding, they are used for the following:
+  - `parse` converts a string to another type (number in this instance) and returns either a value or `Err`;
```

Comment:
`parse` converts a string to another type (a number in this instance) and returns either a value or  an `Err`.